### ISSUE TITLE
Rotated image by default

### ISSUE THREAD

**Issue by chaitu-tk** [aramg/droidcam#36]
Hi, 

I am using droidcam in Ubuntu `x86_64 with 5.3.0-45-generic` kernel. I am able to rotate the image using WebClient, but from a webcam app like cheese/teams/skype the image appears flipped, but the driver only has height and width as module params. If I put it in a landscape mode it works fine, but only in portrait the issue appears, tried the pre-compiled sources and directly compiling from this repo.

Any help is appreciated?

**Comment by aramg** [comment]
Hi,
The video out of Android camera will be landscape, and its up to the client to post-process and  apply rotation/flipping. Unfortunately this isn't available on the Linux client at this time as I never found easy-to-use hardware accelerated libraries to do it.

I have this as a to-do and will hopefully get to in the next little while.

TL;DR: thats not an option right now, you'll need to stick to landscape.

**Comment by aramg** [comment]
Opened #38 to track this.

**Comment by chaitu-tk** [comment]
Okay, thanks. Looking forward to it.

**Comment by DryreL** [comment]
16:9 is ok but where is 9:16 (vertical)?
